### PR TITLE
Ninjectv6 updgrade

### PR DESCRIPTION
**Pull request by WilliamBZA**
Will update the PR once the Ninject package is on MyGet

### CODE REVIEW

**Comment by WilliamBZA** [comment]
Resolves https://github.com/Particular/docs.particular.net/issues/1060#event-538718364
Resolves https://github.com/Particular/docs.particular.net/issues/1059#event-538718500

File: Snippets/Snippets_6/Container/Containers_Ninject.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
+﻿namespace Snippets5.Container
```
Comment (SimonCropp) [inline Snippets/Snippets_6/Container/Containers_Ninject.cs]: fix the namespace

File: samples/containers/ninject/Version_6/Sample/ConfigErrorQueue.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using NServiceBus.Config;
+using NServiceBus.Config.ConfigurationSource;
+
+class ConfigErrorQueue : IProvideConfiguration<MessageForwardingInCaseOfFaultConfig>
```
Comment (SimonCropp) [inline samples/containers/ninject/Version_6/Sample/ConfigErrorQueue.cs]: use SendFailedMessagesTo instead http://docs.particular.net/nservicebus/errors/#configure-your-error-queue-using-code

File: samples/containers/ninject/Version_6/Sample/Program.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using Ninject;
+using NServiceBus;
+
+public class Program
+{
+    static void Main()
+    {
+        new Program().Start().Wait();
```
Comment (SimonCropp) [inline samples/containers/ninject/Version_6/Sample/Program.cs]: we use `.GetAwaiter().GetResult();` everywhere else  https://github.com/Particular/docs.particular.net/blob/master/samples/containers/autofac/Version_6/Sample/Program.cs#L10

File: samples/containers/ninject/Version_6/Sample/Program.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        busConfiguration.UsePersistence<InMemoryPersistence>();
+        busConfiguration.EnableInstallers();
+
+
+        IEndpointInstance endpointInstance = await Endpoint.Start(busConfiguration);
+        await endpointInstance.SendLocal(new MyMessage());
+
+        Console.WriteLine("Press any key to exit");
+        Console.ReadKey();
+        await endpointInstance.Stop();
```
Comment (SimonCropp) [inline samples/containers/ninject/Version_6/Sample/Program.cs]: u want this in a finally https://github.com/Particular/docs.particular.net/blob/master/samples/containers/autofac/Version_6/Sample/Program.cs#L41

**Comment by SimonCropp** [comment]
add a pre-release-marker-file https://github.com/Particular/docs.particular.net#pre-release-marker-file eg https://github.com/Particular/docs.particular.net/blob/master/samples/containers/autofac/Version_6/prerelease.txt

**Comment by WilliamBZA** [comment]
@SimonCropp updated

File: samples/containers/ninject/Version_6/Sample/app.config [hunk 5]
```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" /></startup>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="NServiceBus.Core" publicKeyToken="9fc386479f8a226c" culture="neutral" />
```
Comment (SimonCropp) [inline samples/containers/ninject/Version_6/Sample/app.config]: I suspect this file is redundant

**Comment by WilliamBZA** [comment]
@Simon PR updated. Just need to update the packages version reference once the Ninject PR is merged and published.

File: Snippets/Snippets_6/Snippets_6.csproj [hunk 6]
```diff
@@ -558,6 +579,8 @@
       <Name>Operations</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```
Comment (SimonCropp) [inline Snippets/Snippets_6/Snippets_6.csproj]: Whats this one?

Comment (WilliamBZA) [inline Snippets/Snippets_6/Snippets_6.csproj]: Visual studio adds that for the non-R# test runner. I can remove if wanted, but it's fairly innocuous - and automatic unfortunately.

**Pull request merged**